### DISCUSSION TITLE
Web VSCode webview unable to fetch remote javascript library

### DISCUSSION THREAD

**Discussion by khuongduy354** [microsoft/vscode-discussions#1155] [Extension Development QnA]
Hi, I've followed this tutorial to add javascript to webview [webview script](https://code.visualstudio.com/api/extension-guides/webview#scripts-and-message-passing)

The whole things work on Desktop, or when I run vscode-test-web.

![image](https://github.com/microsoft/vscode-discussions/assets/42113313/cd55fad0-5a5e-4bc2-aac0-5ae52343f8a7)

![image](https://github.com/microsoft/vscode-discussions/assets/42113313/26737ed2-617c-46a1-8f2f-2cb19f0c90b1)

![image](https://github.com/microsoft/vscode-discussions/assets/42113313/44f1ce97-48ad-43f2-91b8-aa8939f3ad08)

But after I published to marketplace with vsce publish, and install it on vscode.dev or github.dev. It show this error:  GET https://<username>.vscode-unpkg.net/path/to/min.js Not Found. 
And the library was unable to load.

**Comment by burkeholland** [comment]
Looks like the base path is different on the web. @isidorn is this a known issue or something on the developer's end.

**Reply by isidorn** [reply]
@khuongduy354 what is the `basePath` in the example you shared?
Make sure to use the VS Code API for path manipulation - the path to your extension should also be exposed through this api.

**Reply by khuongduy354** [reply]
oh, basepath is, context.extensionUri

**Reply by khuongduy354** [reply]
@isidorn  It the same as the tutorial,  when publish the path change to https://.../vscode-unpkg.net...

![image](https://github.com/microsoft/vscode-discussions/assets/42113313/56df1efe-dcff-4304-8717-82e0d366dc8d)

**Reply by burkeholland** [reply]
Were you able to resolve this issue?

**Comment by olguzzar** [comment]
Closing this discussion as it's been a while without any updates - please feel free to reopen if you have new information!
